lib/spack/env/cc

```diff
@@ -98,25 +98,25 @@ case "$command" in
     cpp)
         mode=cpp
         ;;
-    cc|c89|c99|gcc|clang|icc|pgcc|xlc)
+    cc|c89|c99|gcc|clang|icc|pgcc|xlc|xlc_r)
```

Comment:
@serbanmaerean: you can actually do this:

```yaml
- compiler:
    environment: {}
    extra_rpaths: []
    flags: {}
    modules: []
    operating_system: rhel7
    paths:
      cc: /usr/bin/xlc_r
      cxx: /usr/bin/xlc++_r
      f77: /usr/bin/xlf_r
      fc: /usr/bin/xlf2003_r
    flags:
      fflags: -qzerosize -qfixed
    spec: xl_r@13.1
    target: ppc64le
```

That stuff just needs to go in the `flags` section.  If that doesn't work let me know.  Do these packages *need* those options to compile?  If so the flags should probably go in the package file, or at least be exposed as variants.